src/georeferencing/pixel_to_latlong.py

```
@@ -99,5 +109,8 @@ def _compute_homography(pixels, world_coords):
     pixels = np.array(pixels, dtype=np.float32)
     world_coords = np.array(world_coords, dtype=np.float32)
 
-    H, _ = cv2.findHomography(pixels, world_coords, method=cv2.RANSAC)
-    return H
+    try:
```

same here - we should never return None - if something false, we should raise an exception 

but it's only if we want to handle the case, it's also okay if we revert to the original and you change the tests. as you'd like